pkg/api/team_members.go

```diff
@@ -92,27 +96,28 @@ func (hs *HTTPServer) UpdateTeamMember(c *models.ReqContext) response.Response {
 	if err != nil {
 		return response.Error(http.StatusBadRequest, "teamId is invalid", err)
 	}
-	orgId := c.OrgId
-
-	if err := hs.teamGuardian.CanAdmin(c.Req.Context(), orgId, teamId, c.SignedInUser); err != nil {
-		return response.Error(403, "Not allowed to update team member", err)
+	userId, err := strconv.ParseInt(web.Params(c.Req)[":userId"], 10, 64)
+	if err != nil {
+		return response.Error(http.StatusBadRequest, "userId is invalid", err)
 	}
+	orgId := c.OrgId
 
-	if c.OrgRole != models.ROLE_ADMIN {
```

Comment:
The check for whether the last admin isn't being removed is now done directly from [team SQL store](https://github.com/grafana/grafana/pull/43991/files#diff-9ff4b79076656e607d76f40213e6a3cff3e4bbf1ec0deda50dee277b25ebaae3R372).